test/couchdb_views_tests.erl

```diff
@@ -286,7 +286,13 @@ couchdb_1309(DbName) ->
         ?assert(is_pid(NewIndexerPid)),
         ?assert(is_process_alive(NewIndexerPid)),
         ?assertNotEqual(IndexerPid, NewIndexerPid),
-        ?assertEqual(2, count_users(DbName)),
+        UserCnt = case count_users(DbName) of
```

Comment:
so I suggest pulling this retry out into a function with a countdown parameter, as long as its finite retry we should fix both issues.